File: ACP_comm_test/ACP/implementations/java/ACPSender.java
/**
 * ACPSender.java
 * 
 * This file is an example of the acpcomms python Messenger class showcasing 
 * UDP streaming
 * 
 * This is meant to be used with ACPReceiver.java
 */

import ACP.acpcomms.java.Streamer;

import java.util.HashMap;
import java.util.Map;

public class ACPSender {
    public static void main(String[] args) throws Exception {
        Streamer sender = new Streamer();
        
        Map<String, Object> config = new HashMap<>();
        config.put("host", "localhost");
        config.put("port", 9999);
        
        sender.configure(config);
        sender.connect();
        
        // Send data
        for (int i = 0; i < 10; i++) {
            sender.sendData("UDP message");
            Thread.sleep(2000);
        }
        
        sender.disconnect();
    }
}

File: ACP_comm_test/ACP/implementations/python/acp_sender.py
# acp_sender.py
# 
# This file is an example of the acpcomms python Streamer class showcasing UDP 
# streaming
# 
# This is meant to be used with acp_receiver.py

from ACP.acpcomms.python3.streamer import Streamer
import time

sender = Streamer()
sender.configure({
    "host": "localhost",
    "port": 9999
})
sender.connect()

# Send data
for i in range(10):
    sender.sendDataString("UDP message")
    time.sleep(2)
sender.disconnect()